### ISSUE TITLE
type-assertions: assert is not defined

### ISSUE THREAD

**Issue by pflannery** [google/traceur-compiler#1717]
When trying to use type assertions I get an error ```ReferenceError: assert is not defined```. 

[REPL Example](http://google.github.io/traceur-compiler/demo/repl.html#%2F%2F%20Options%3A%20--member-variables%20--type-assertions%20--source-maps%20--types%20%0A%0Aclass%20Test%20%7B%0A%20%20constructor(value%20%3A%20number)%20%7B%0A%20%20%09this.item%20%3D%20value%3B%0A%20%20%7D%0A%7D%0A%0Avar%20p%20%3D%20new%20Test(123)%3B)

offending line:
```js
assert.argumentTypes(value, $traceurRuntime.type.number);
```

**Comment by arv** [comment]
The initial design was to make the type assertion library user configurable. That was a bad idea and I want these asserts to be moved to traceur runtime.

**Comment by pflannery** [comment]
:+1:

**Comment by SerkanSipahi** [comment]
@arv is the assert lib moved to traceur runtime? i believe not, see here: [REPL](http://google.github.io/traceur-compiler/demo/repl.html#%2F%2F%20Options%3A%20--annotations%20--array-comprehension%20--async-functions%20--async-generators%20--exponentiation%20--for-on%20--generator-comprehension%20--member-variables%20--proper-tail-calls%20--require%20--source-maps%20--symbols%20--type-assertions%20--types%20%0Aclass%20Selector%20%7B%0A%20%20constructor(nodes%20%3A%20List%3CElement%3E)%20%3A%20List%3CElement%3E%20%7B%0A%0A%20%20%7D%0A%7D%0A%0Anew%20Selector(document.querySelectorAll('div'))%3B)

1.) is there any release date for moving? 
2.) do yo have recommendation which assert lib we can use in the meantime with traceur runtime ?

**Comment by arv** [comment]
1. No plan.

2. Use the one that is in the tests. That one has the right semantics.

**Comment by SerkanSipahi** [comment]
you mean this one: https://github.com/chaijs/chai

**Comment by arv** [comment]
Not entirely... 

 https://github.com/chaijs/chai in combination with https://github.com/google/traceur-compiler/blob/master/test/feature/TypeAssertions/resources/assert.js

However, we should just move this to runtime and remove the dependency on chai.

**Comment by SerkanSipahi** [comment]
right, there I found the file, too https://github.com/google/traceur-compiler/blob/master/test/feature/TypeAssertions/resources/assert.js
